x/orderbook/msg.go

```diff
@@ -34,60 +36,58 @@ func (CancelOrderMsg) Path() string {
 }
 
 func (m CreateOrderBookMsg) Validate() error {
-	if err := m.Metadata.Validate(); err != nil {
-		return errors.Wrap(err, "metadata")
-	}
-	if err := validID(m.MarketID); err != nil {
-		return errors.Wrap(err, "market id")
-	}
+	var errs error
+
+	errs = errors.AppendField(errs, "metadata", m.Metadata.Validate())
+	errs = errors.AppendField(errs, "market id", validID(m.MarketID))
```

Comment:
Field name should follow Go naming [as documented](https://godoc.org/github.com/iov-one/weave/errors#Field). Please change it to `Metadata` and `MarketID`.